doc/man1/flux-account-view-job-records.rst

```diff
@@ -63,6 +63,13 @@ job records.
     column by. Expressions can be started with any of the following operators:
     ``<``, ``<=``, ``=``, ``>=``, or ``>``.
 
+.. option:: -D/--duration-delta
+
+    Return jobs that fit a certain duration delta's criteria by passing an
+    expression. Multiple filters can be passed to refine the ``duration_delta``
+    column by. Expressions can be started with any of the following operators:
+    ``<``, ``<=``, ``=``, ``>=``, or ``>``.
+
```

Comment:
It might be nice to provide an example here?

Reply:
Great suggestion! I went ahead and force-pushed up a change to add a short example at the bottom of the man page.